salt/utils/hashutils.py

```diff
@@ -82,6 +82,16 @@ def md5_digest(instr):
     return hashlib.md5(instr).hexdigest()
 
 
+def sha1_digest(instr):
+    '''
+    Generate an sha1 hash of a given string.
```

Comment:
Might want to add a comment "sha1 is not consider secure, however an acceptable use is for generating fix sized filenames". Maybe md5 should have the same comment as well.   Just a thought.